kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx

```diff
@@ -166,5 +166,24 @@ describe('BrokersList Component', () => {
         );
       });
     });
+
+    describe('when some brokers have no diskUsage', () => {
+      beforeEach(() => {
+        (useBrokers as jest.Mock).mockImplementation(() => ({
+          data: brokersPayload,
+        }));
+        (useClusterStats as jest.Mock).mockImplementation(() => ({
+          data: {
+            ...clusterStatsPayload,
+            diskUsage: [clusterStatsPayload.diskUsage[0]],
+          },
+        }));
+      });
+      it('renders list of all brokers', async () => {
+        renderComponent();
+        expect(screen.getByRole('table')).toBeInTheDocument();
+        expect(screen.getAllByRole('row').length).toEqual(3);
```

Comment:
can we make the  `.toEqual(3);` the number inside to be dynamic based on the mock data ?

Reply:
Sure, done